spec/src/main/asciidoc/metrics.asciidoc

```diff
@@ -0,0 +1,177 @@
+//
+// Copyright (c) 2018 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// Contributors:
+// Andrew Rouse
+
+== Integration with Microprofile Metrics
```

Comment:
As we clarified on the hangout that all Counters are monotonic and we're going to remove `_total` suffix we need to state this also in the spec. The Metrics counters can be monotonic but can also be decremented, see [Counted](https://github.com/eclipse/microprofile-metrics/blob/master/api/src/main/java/org/eclipse/microprofile/metrics/annotation/Counted.java#L105).

I suggest to replace "Counter" in every metric type with "Monotonic Counter"

Reply:
mpMetrics doesn't currently add `_total` to the end of counters -- I'd suggest we indicate that counter names should end in `_total`.  If mpMetrics adds `_total` to the end of counter names in prometheus format it should avoid doing so to metric names that already end in _total.

Reply:
There's already a note at the bottom that all the counters are monotonic. As we're now keeping the `total` suffix, I think this is sufficient.

I'm reluctant to put "Monotonic Counter" in the type column of the table as `Counter` is the interface in the Metrics API which must be used.